### PR TITLE
add fave104

### PR DESCRIPTION
**Pull request by duilchoi**
Add Linworks fave104 firmware

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/linworks/fave104/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/linworks/fave104/keymaps/default/keymap.c:1]: Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment (duilchoi) [reply keyboards/linworks/fave104/keymaps/default/keymap.c:1]: It was deleted by my mistake. I'll try to fix it.

File: keyboards/linworks/fave104/info.json [hunk 2]
```diff
@@ -0,0 +1,376 @@
+{
+    "keyboard_name": "fave104",
+    "url": "",
+    "maintainer": "keydler",
```
Comment (fauxpark) [inline keyboards/linworks/fave104/info.json:4]:
```suggestion
    "keyboard_name": "FAVE-104",
    "manufacturer": "Lx3",
    "url": "",
    "maintainer": "keydler",
    "usb": {
        "vid": "0x4C58",
        "pid": "0x000A",
        "device_version": "0.0.1"
    },
```

File: keyboards/linworks/fave104/config.h [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4C58 //"LX"
+#define PRODUCT_ID      0x000A // FAve 104
+#define DEVICE_VER      0x0001 // Version 1
+#define MANUFACTURER    Lx3
+#define PRODUCT        FAVE-104
+
```
Comment (fauxpark) [inline keyboards/linworks/fave104/config.h:28]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**